src/components/tx/TxData.js

```diff
@@ -933,6 +1077,129 @@ class TxData extends React.Component {
+
+            {!hathorLib.transactionUtils.isBlock(this.props.transaction) && renderFirstBlockDiv()}
+            <div className="summary-balance-info-container">
+              <div className="address-container-title">Weight:</div>{' '}
+              {helpers.roundFloat(this.props.transaction.weight)}
+            </div>
+            {this.props.transaction.signer_id && (
+              <div className="summary-balance-info-container">
+                <div className="address-container-title">Signer ID:</div>{' '}
+                {this.props.transaction.signer_id.toLowerCase()}
+              </div>
+            )}
+            {this.props.transaction.signer && (
+              <div className="summary-balance-info-container">
+                <div className="address-container-title">Signer:</div>{' '}
+                {helpers.getShortHash(this.props.transaction.signer.toLowerCase())}
+              </div>
+            )}
+
+            {hathorLib.transactionUtils.isBlock(this.props.transaction) && renderHeight()}
+            {hathorLib.transactionUtils.isBlock(this.props.transaction) && renderScore()}
+            {!hathorLib.transactionUtils.isBlock(this.props.transaction) && renderAccWeightDiv()}
+            {!hathorLib.transactionUtils.isBlock(this.props.transaction) &&
+              renderConfirmationLevel()}
+          </div>
+          <div className="details-container-gap">
+            {this.props.transaction.version === hathorLib.constants.NANO_CONTRACTS_VERSION && (
+              <div className="d-flex flex-row align-items-start mb-3">{renderNCData()}</div>
+            )}
+
+            {this.props.transaction.version === hathorLib.constants.NANO_CONTRACTS_VERSION && (
+              <div className="d-flex flex-row align-items-start mb-3"> {renderNCActions()}</div>
+            )}
+
+            <div className="tx-drop-container-div">
+              <DropDetails startOpen title={`Inputs (${this.props.transaction.inputs.length})`}>
+                {renderInputs(this.props.transaction.inputs)}
+              </DropDetails>
+              <DropDetails startOpen title={`Outputs (${this.props.transaction.outputs.length})`}>
+                {renderOutputs(this.props.transaction.outputs)}
+              </DropDetails>
+            </div>
+            {/* {this.state.tokens.length > 0 && renderTokenList()} */}
+            <div className="tx-drop-container-div">
+              <DropDetails title="Parents:">
+                {renderNewUiDivList(this.props.transaction.parents)}
+              </DropDetails>
+              <DropDetails title="Children:">
+                {renderNewUiDivList(this.props.meta.children)}
+              </DropDetails>
+            </div>
+            {this.state.graphs.map((graph, index) => renderNewUiGraph(index))}
+
+            {hathorLib.transactionUtils.isBlock(this.props.transaction) && (
+              <DropDetails title="Feature Activation:" onT={e => this.toggleFeatureActivation(e)}>
```

Comment:
issue: On the new UI, when clicking the "Feature Activation" details button, I receive the following error. [Example block](http://localhost:3000/transaction/0000000000927dcece85bfb8a1284819c13795650ba37cc5ff15d6e7f1da8833) that I used.
```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'preventDefault')
    at push../src/components/tx/TxData.js.constructor.toggleFeatureActivation (TxData.js:194:1)
    at Object.onT (TxData.js:1171:1)
    at click (DropDetails.js:9:1)
```

Reply:
corrected. Waiting Review.🙂